javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,22 @@
+var $spans = document.querySelectorAll('span');
+var i = 0;
+
+function typingEvent(event) {
+  if (!$spans[i]) {
+    if (i === 0) {
+      $spans[i].className = 'active';
+    }
```

Comment:
The condition on line 6 won't be true, given that `$spans[i]` doesn't exist. So you can remove this whole 6-8